saleor/site/models.py

```diff
@@ -22,6 +22,12 @@ class SiteSettings(models.Model):
     include_taxes_in_prices = models.BooleanField(default=True)
     display_gross_prices = models.BooleanField(default=True)
     charge_taxes_on_shipping = models.BooleanField(default=True)
+    handle_stock_by_default = models.BooleanField(
+        default=True,
+        help_text=pgettext_lazy(
```

Comment:
Again, this help_text should be moved to the form. I think we need to explain precisely what this feature does, in a user-friendly way. I'm thinking about a checkbox with descriptive label and help_text:
- label: "Enable inventory tracking for newly created products."
- help text: "Inventory tracking may be modified at the product-level. Changing this setting won't affect existing products."

What do you think?